cdap-ui/app/features/adapters/templates/create/tabs/edit-transform-properties.html

```diff
@@ -1,7 +1,7 @@
 <br />
 
 <div class="row">
-  <div class="col-xs-12" ng-init="plugin=transform">
+  <div class="col-xs-12" ng-init="plugin=transform; type=AdapterCreateController.model.metadata.type">
```

Comment:
hmmm after further inspection, I think we can delete the 3 files:
1. edit-sink-properties.html
2. edit-source-properties.html
3. edit-transform-properties.html

Am I missing anything?